src/main/java/com/bfrisco/benchants/enchants/Durability.java

```diff
@@ -25,7 +27,16 @@ public Durability() {
     @EventHandler
     @SuppressWarnings("unused")
     public void onPlayerItemDamageEvent(PlayerItemDamageEvent event) {
-        if (!hasDurability(event.getItem())) return;
+
+        Player player = event.getPlayer();
+        ItemStack item = event.getPlayer().getInventory().getItemInMainHand();
+//*****************Proposed new check, removes nbtapi dependency*********************************
+        if (!ItemInfo.isTitanTool(item)) return;
+        if (!ItemInfo.isActive(item)) return;
+//*****************Proposed new check, removes nbtapi dependency*********************************
```

Comment:
This looks good - at this point we should start being able to remove the NBT imports, and the NBT dependency altogether.

We can probably just remove the `/benchants apply` command instead and opt for just creating the titan tool.

If you want to work on this, go ahead - but if not do not worry about removing the NBT dependency for now